vim25/types/helpers.go

```diff
@@ -316,6 +322,42 @@ func (ci VirtualMachineConfigInfo) ToConfigSpec() VirtualMachineConfigSpec {
 	return cs
 }
 
+type ByteArray []byte
```

Comment:
A comment on the exported symbol describing its purpose and the behavior of the XML encoding/decoding logic would be nice.

Reply:
Agreed, meant to, done.